armi/reactor/tests/test_blocks.py

```diff
@@ -1201,6 +1201,20 @@ def test_adjustDensity(self):
 
         self.assertAlmostEqual(mass2 - mass1, massDiff)
 
+    @patch.object(blocks.HexBlock, "getSymmetryFactor")
```

Comment:
Shouldn't we also have a test where the symmetry factor is 1, so we can see the usual case wasn't broken?

Reply:
I added a case to the test with the symmetry factor at 1